apiv2/user/user.go

```diff
@@ -98,6 +99,28 @@ func (c *RESTClient) GetUser(ctx context.Context, username string) (*model.User,
 	return nil, &ErrUserNotFound{}
 }
 
+// GetUserByID fetches a registered user by the provided user id.
+// Returns an error if no user could be found, or if the id is '0'.
+func (c *RESTClient) GetUserByID(ctx context.Context, id int64) (*model.User, error) {
+	if id == 0 {
```

Comment:
what about IDs that are less than 0?
```suggestion
	if id <= 0 {
```

Reply:
Good catch :+1: Added this + test-case in https://github.com/mittwald/goharbor-client/pull/88/commits/b330a423f084e4d81cd0afb19c43a9f0de940b61